app/Http/Controllers/Chart/SectorChartController.php

```diff
@@ -81,6 +81,8 @@ function gradeChart($id){
                         [
                             'ticks' => [
                                 'beginAtZero' => true,
+                                'stepSize' => 1,
+                                'step' => 1,
```

Comment:
Can you show me a screenshot for this change ?

Reply:
https://github.com/lucien-chastan/oblyk/pull/27 here you have cleaner pull request with links to screenshot of this change.